canonical/jaas-dashboard
# Truncate long usernames in the access panel.

## Done

- Truncate long usernames in the access panel.
- Migrate tests to RTL.

## QA

- Go to the model list.
- Open the access panel and add a user with a really long name (so that it doesn't fit in the box).
- You should see the username truncated.
- Hover the username and you should see the full name in the tooltip.

## Details

Fixes: #1245.

## Screenshots

<img width="377" alt="Screenshot 2022-10-26 at 11 12 33 am" src="https://user-images.githubusercontent.com/361637/197907145-63fcb4e6-5ffa-485a-a849-c6efc2081821.png">
